playbooks/robusta_playbooks/k8s_resource_enrichments.py

```diff
@@ -0,0 +1,52 @@
+from robusta.api import *
+
+supported_resources = [
+    "Deployment",
+    "DaemonSet",
+    "ReplicaSet",
+    "Pod",
+    "StatefulSet"
+]
+
+
+@action
+def related_pods(event: KubernetesResourceEvent):
+    """
+    Return the list of pods related to that k8s resource.
+    For example, return all pods of a given k8s Deployment
+
+    Supports Deployments, ReplicaSets, DaemonSets, StatefulSets and Pods
+    """
+    resource = event.get_resource()
+    if resource.kind not in supported_resources:
+        logging.error(f"Related pods is not supported for resource {resource.kind}")
+        return
+
+    if resource.kind == "Pod":
+        pods = [resource]
+    else:
+        selector = build_selector_query(resource.spec.selector)
+        pods = PodList.listNamespacedPod(namespace=resource.metadata.namespace, label_selector=selector).obj.items
+
+    rows = []
+    for pod in pods:
```

Comment:
looks good, i personally would wrap everything in a function `generate_related_pod_row` but its fine like this

Reply:
fixed